docs/collections.md

```diff
@@ -10,24 +10,21 @@ Then run the following command:
 ./createcollection -c <collection_name>
 ```
 
-The script will ask you to run additional commands if it ran succesfully:
-```shell
-# run migrations
-docker-compose run --rm snoop--<collection_name> ./manage.py migrate
-# reset stats index
-docker-compose run --rm snoop--<collection_name> ./manage.py resetstatsindex
-# create collection inside the snoop docker image
-docker-compose run --rm snoop--<collection_name> ./manage.py createcollection <collection_name> /opt/hoover/collections/<collection_name>/data
-# add collection to search
-docker-compose run --rm search ./manage.py addcollection <collection_name> http://snoop--<collection_name>/collections/<collection_name>/json --public
-# add ocr source
-docker-compose run --rm snoop--<collection_name> ./manage.py createocrsource ocr-name /opt/hoover/collections/<collection_name>/ocr-data
-```
+The script will ask you to run additional commands if it ran successfully:
+
+1. Start added containers using docker-compose
+` $ docker-compose up -d`
```

Comment:
I think you intended to have the code example on the next line. To achieve that, wrap it in triple backticks, and indent the whole thing with 4 spaces.

Reply:
That doesn't change anything in the Md-viewers that I tested. And it looks like I intended.
<img width="739" alt="collection docs line 15" src="https://user-images.githubusercontent.com/42345537/51461022-5531c580-1d5d-11e9-9211-2eafab2c74d9.png">

Reply:
Ah, I'm looking at [what GitHub renders](https://github.com/hoover/docker-setup/blob/f873efbfc4cef1ffae12bbe4beeb0dbec0cca858/docs/collections.md). Weird. You can try [grip](https://github.com/joeyespo/grip) for github-specific markdown preview. Anyway, here's how I'd write it:

```markdown
1. start hoover again and spin up the containers for the new collection
    ```
    $ docker-compose up -d`
    ```

2. wait for the database
    ```
    $ docker-compose run --rm snoop--<collection_name> /wait`
    ```

3. create the data model in the snoop container
    ```
    $ docker-compose run --rm snoop--<collection_name> ./manage.py migrate`
    ```

4. import the index
    ```
    $ docker-compose run -T  --rm snoop--<collection_name> ./manage.py importindex < <collection_name>-index.tgz`
    ```

5. add the collection to search
    ```
    $ docker-compose run --rm search ./manage.py addcollection <collection_name>--index <collection_name> http://snoop--<collection_name>/collection/json --public`
    ```
```